### PR TITLE
add mount point json pattern description

### LINKED ISSUE #1995: Readme is missing some information about connection formats
In the section [here](https://github.com/quasar-analytics/quasar#database-mounts) there's a full example of a MongoDB mount, but for the other connectors on the connection URI is listed, and not the mount type name used as the key above the `connectionUri`:

```
      "mongodb": {
        "connectionUri": "mongodb://<user>:<pass>@<host>:<port>/<dbname>"
      }
```

Can we get some clarification on what these values should be? We're assuming `mongodb`, `marklogic`, `couchbase`, but we've discovered some confusion about the name for `spark` in the code we have in the frontend currently (slamdata/purescript-quasar#27).

### PR DESCRIPTION
**Pull request by nicflores**
Fixes #1995.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,13 +123,31 @@ To run the JAR, execute the following command:
...
 ```
+The `<mountPath>` is the name of your mount point and the remaining parameters are listed below:
+
+| mountKey        | protocol         | uri                                    |
+|-----------------|------------------|----------------------------------------|
+| `couchbase`     | `couchbase://`   | [Couchbase](#couchbase)                |
+| `marklogic`     | `xcc://`         | [MarkLogic](#marklogic)                |
+| `mongodb`       | `mongodb://`     | [MongoDB](#database-mounts)            |
+| `spark-hdfs`    | `spark://`       | [Spark HDFS](#hdfs-using-apache-spark) |
+| `spark-local`   | `spark_local=`   | [Spark](#hdfs-using-apache-spark)      |
```
Comment (garyb) [inline README.md]: This links to the same thing as Spark HDFS - although actually there doesn't seem to be a section for Spark local, so there's nothing to link to!

Similarly there's a Spark FTP now too I think. Perhaps these things are out of scope for this PR though, I could just open another issue for those, for @rabbitonweb to deal with perhaps?

**Comment by drostron** [comment]
Don't forget about `[skip ci]` for future readme updates.

File: README.md [hunk 2]
```diff
@@ -123,13 +123,31 @@ To run the JAR, execute the following command:
...
 
-As a command-line REPL user, to work with a fully functioning REPL you will need the metadata store and a mount point. See [here](#full-testing-prerequisite-docker-and-docker-compose) for instructions on creating the metadata store backend using docker. To add a mount you can start the web server mentioned [below](#web-jar) and issue a `curl` command like:
+As a command-line REPL user, to work with a fully functioning REPL you will need the metadata store and a mount point. See [here](#full-testing-prerequisite-docker-and-docker-compose) for instructions on creating the metadata store backend using docker.
+
+Once you have a running metastore you can start the web api service with [these](#web-jar) instructions and issue curl commands 
+of the following format to create new mount points.
 
 ```bash
-curl -v -X PUT http://localhost:8080/mount/fs/cb/ -d '{ "couchbase": { "connectionUri":"couchbase://192.168.99.100/beer-sample?password=&docTypeKey=type" } }'
+curl -v -X PUT http://localhost:8080/mount/fs/<mountPath>/ -d '{ "<mountKey>": { "connectionUri":"<protocol://><uri>" } }'
```
Comment (drostron) [inline README.md]: You'll want to leave '://' off protocol here.

File: README.md [hunk 3]
```diff
@@ -123,13 +123,31 @@ To run the JAR, execute the following command:
...
 ```
+The `<mountPath>` is the name of your mount point and the remaining parameters are listed below:
```
Comment (drostron) [inline README.md]: s/is the name/specifies the path/

File: README.md (hunk 1 above)
Comment (nicflores) [inline README.md]: Let's see what @rabbitonweb has to say. I can easily add and section to Spark explaining the `spark_local=` protocol but I'm not too sure about the FTP, maybe docs for that are already in the works.

File: README.md [hunk 4]
```diff
@@ -247,13 +265,19 @@ Known Limitations
...
-#### HDFS using Apache Spark
+#### Apache Spark
+
+To connect to Apache Spark and use either local files or HDFS to query data use the following `connectionUri`:
+
+with local files:
+
+`spark_local=\"/path/to/data/my.data\"`
 
-To connect to HDFS using Apache Spark use the following `connectionUri` format:
```
Comment (rabbitonweb) [inline README.md]: this is not correct, correct URI is `spark:///<host>:<port>?rootPath=<rootPath>&hdfsUri=<hdfsUri>[&spark_configuration=spark_configuration_value]`

Comment (nicflores) [inline README.md]: Thanks @rabbitonweb, I'll update it to the correct uri.

Comment (drostron) [inline README.md]: @rabbitonweb Two slashes not tree right?

Comment (nicflores) [inline README.md]: change requests updated

File: README.md [hunk 5]
```diff
@@ -123,13 +123,31 @@ To run the JAR, execute the following command:
...
+As a command-line REPL user, to work with a fully functioning REPL you will need the metadata store and a mount point. See [here](#full-testing-prerequisite-docker-and-docker-compose) for instructions on creating the metadata store backend using docker.
+
+Once you have a running metastore you can start the web api service with [these](#web-jar) instructions and issue curl commands 
+of the following format to create new mount points.
 
 ```bash
-curl -v -X PUT http://localhost:8080/mount/fs/cb/ -d '{ "couchbase": { "connectionUri":"couchbase://192.168.99.100/beer-sample?password=&docTypeKey=type" } }'
+curl -v -X PUT http://localhost:8080/mount/fs/<mountPath>/ -d '{ "<mountKey>": { "connectionUri":"<protocol><uri>" } }'
 ```
+The `<mountPath>` is the name of your mount point and the remaining parameters are listed below:
```
Comment (drostron) [inline README.md]: s/is the name/specifies the path/

**Comment by drostron** [comment]
@nicflores Just a small update request, then this is ready.

**Comment by nicflores** [comment]
@drostron, thanks for the reminder. fix pushed.
